elements/cc-common/static/etc/custom-motd/05-lease-info

```diff
@@ -0,0 +1,67 @@
+#!/bin/bash
+
+REGION=$(cc-read-vendordata "region")
+if [ "${REGION}" = "KVM@TACC" ]; then
+  exit 0
+fi
+
+set -euo pipefail
+
+error_out() {
+    echo -e "    ERROR: $1\n" >&2
+    echo -e "    Unable to gather lease information for this instance\n"
+    exit 1
+}
+
+echo -e "\nLEASE INFORMATION"
+
+TMP_OUT=$(mktemp) || error_out "Failed to create temp file for lease info"
+TMP_ERR=$(mktemp) || error_out "Failed to create temp file for lease info"
+
+NODE_ID=$(cc-read-vendordata "node")
+
+if [ ! -f /home/cc/openrc ]; then
+  OPENRC=$(cc-read-vendordata "openrc")
+  if [ "${OPENRC}" != "null" ]; then
+    cc-read-vendordata "openrc" >  /home/cc/openrc
+    sed -i 's/^ *//g' /home/cc/openrc
+  else
+    error_out "Unable to read OPENRC"
+  fi
+fi
+
+source /home/cc/openrc
+if ! openstack reservation host show "${NODE_ID}" -f json >"$TMP_OUT" 2>"$TMP_ERR"; then
+    error_out "Failed to get host info: $(cat "$TMP_ERR")"
+fi
+BLAZAR_NODE_ID=$(jq -r '.id' <"$TMP_OUT" || error_out "Invalid host JSON response")
+
+if ! openstack reservation host allocation show "$BLAZAR_NODE_ID" -f json >"$TMP_OUT" 2>"$TMP_ERR"; then
+    error_out "Failed to get host allocation info: $(cat "$TMP_ERR")"
+fi
+RESERVATION_INFO=$(cat "$TMP_OUT")
+
+LEASE_ID=$(echo "$RESERVATION_INFO" | jq -e -r '.reservations | fromjson | .lease_id' || error_out "Invalid lease data in response")
+START_DATE=$(echo "$RESERVATION_INFO" | jq -e -r '.reservations | fromjson | .start_date' || error_out "Missing start date in response")
+END_DATE=$(echo "$RESERVATION_INFO" | jq -e -r '.reservations | fromjson | .end_date' || error_out "Missing end date in response")
```

Comment:
is blazar returning a single reservation here, or an array of them? 

You might want to test when there are "future" pending reservations for the same node

Reply:
Good call. I'll check/fix that.

Reply:
Or better yet: fix blazar's json, but also update vendordata to give back the information we need here (like you mentioned on trello) so we don't have to parse it. Maybe we should make that a future/soon ticket to do and then we can just drop this approach.

Reply:
this bit of the blazarclient we can probably fix without disrupting much, it's already something we've modified extensively.

but, the issue is blazar returning nested json, so the following hack with jq might get us unstuck in the meantime.
```
openstack reservation host allocation show \
    $RESERVATION_HOST_ID -f json \
    | jq -r '.reservations'  \
    | jq -s '.'
```

Reply:
I'm not sure I follow the bit about nested json. I'd expect nested json to still be processable, which this json from blazar isn't (even though I'd expect it to be with explicit json formatting). But agreed using jq to slurp those individual json entries together into a list is more straightforward than rewriting the string from blazar so I'll switch to that.